pkg/virt-handler/cgroup/cgroup.go

```diff
@@ -138,6 +156,28 @@ func getCpuSetPath(manager Manager, cpusetFile string) (string, error) {
 	return cpusetStr, nil
 }
 
+// set cpus "cpusList" on the allowed CPUs. Optionally on a subcgroup of
+// the pods control group (if subcgroup != nil).
+func SetCpuSet(manager Manager, subcgroup string, cpusList []int) error {
+	subsyspath, err := manager.GetBasePathToHostSubsystem("cpuset")
+	if err != nil {
+		return err
+	}
+
+	if subcgroup != "" {
+		subsyspath = filepath.Join(subsyspath, subcgroup)
+	}
+
+	wval := strings.Trim(strings.Replace(fmt.Sprint(cpusList), " ", ",", -1), "[]")
+
+	err = runc_cgroups.WriteFile(subsyspath, "cpuset.cpus", wval)
```

Comment:
you can return the value from `runc_cgroups`,  no need for the next lines.

Reply:
Fixed.